addon/components/frost-select.js

```diff
@@ -345,7 +345,9 @@ export default Component.extend({
       const onInput = this.get('onInput')
 
       if (typeOf(onInput) === 'function') {
-        onInput(filter)
+        run.later(() => {
```

Comment:
It looks like using `run.next` instead actually causes some tests to fail where `run.later` doesn't.